File: Greedy/python/01_거스름돈.py
# fast campus 강의
# https://www.acmicpc.net/problem/5585
# 1

changes = 1000 - int(input())
count = 0

for i in [500, 100, 50, 10, 5, 1]:
    count += changes // i
    changes %= i

print(count)

File: backup/Greedy/c++/01_거스름돈.cpp
// fast campuse 강의
// https://www.acmicpc.net/problem/5585
// 2

#include <iostream>

using namespace std;

int main()
{
    int change, result = 0;
    int money[6] = {500, 100, 50, 10, 5, 1};
    cin >> change;
    change = 1000 - change;
    for (int i = 0; i < 6; i++) {
        result += change / money[i];
        change %= money[i];
    }
    cout << result;
}